src/loader.ts

```diff
@@ -42,29 +49,43 @@ export default class {
      */
     async initConfigTypes() {
         const filename = basename(this.config.importPath);
+        
         mkdirSync(`${this.cwd}/@types/${filename}`, { recursive: true });
         Bun.write(`${this.cwd}/@types/${filename}/lastModified`, lastModified(this.config.importPath));
+        
         const configWriter = Bun.file(`${this.cwd}/@types/${filename}/config.ts`).writer();
-        configWriter.write(`import { LoaderConfig, T } from "hyperimport";\nexport default {\n\tbuildCommand: ${JSON.stringify(this.config.buildCommand)},\n\toutDir: "${this.config.outDir}",\n\tsymbols: {`);
-        for (const symbol of nm(this.config.libPath)) {
+        
+        configWriter.write(
+            `import { LoaderConfig, T } from "hyperimport";\n` 
+            + `export default {\n\tbuildCommand: ${JSON.stringify(this.config.buildCommand)},\n\toutDir: "${this.config.outDir}",\n\tsymbols: {`
+        );
+        
+        for (const symbol of nm(this.config.libPath)) 
             configWriter.write(`\n\t\t${symbol}: {\n\t\t\targs: [],\n\t\t\treturns: T.void\n\t\t},`);
-        }
+        
         configWriter.write(`\n\t}\n} satisfies LoaderConfig.Main;`);
         configWriter.end();
+        
         Bun.write(
             `${this.cwd}/@types/${filename}/types.d.ts`,
             `declare module "*/${filename}" {\n\tconst symbols: import("bun:ffi").ConvertFns<typeof import("./config.ts").default.symbols>;\n\texport = symbols;\n}`
         );
-        console.log(`\n\x1b[32mConfig file has been generated at "${this.cwd}/@types/${filename}/config.ts"\x1b[39m\nEdit the config.ts and set the argument and return types, then rerun the script.`);
+        
+        console.log(
+            `\n\x1b[32mConfig file has been generated at "${this.cwd}/@types/${filename}/config.ts"\x1b[39m\n` 
+            + `Edit the config.ts and set the argument and return types, then rerun the script.`
+        );
     }
 
     /**
      * When the source file isn't configured yet, this executes to configure it.
      */
     async initConfig() {
         await this.initConfigPre();
+        
         console.log("\nBuilding the source file...");
         await this.build();
+        
```

Comment:
I am opposed for this new line here too, the source file build success message should be together with them.